pyvista/core/filters.py

```diff
@@ -3367,3 +3367,48 @@ def delaunay_2d(ugrid, tol=1e-05, alpha=0.0, offset=1.0, bound=False):
         return pyvista.PolyData(ugrid.points).delaunay_2d(tol=tol, alpha=alpha,
                                                           offset=offset,
                                                           bound=bound)
+
+
+class UniformGridFilters(DataSetFilters):
+
+    def __new__(cls, *args, **kwargs):
+        if cls is UniformGridFilters:
+            raise TypeError("pyvista.UniformGridFilters is an abstract class and may not be instantiated.")
+        return object.__new__(cls)
+
+    def gaussian_smooth(dataset, radius_factor=1.5, std_dev=2.,
+                        scalars=None, preference='points'):
+        """Smooths the data with a Gaussian kernel
+
+        Parameters
+        ----------
+        radius_factor : float or iterable, optional
+            Unitless factor to limit the extent of the kernel.
+
+        std_dev : float or iterable, optional
+            Standard deviation of the kernel in pixel units.
+
+        scalars : str, optional
+            Name of scalars to process. Defaults to currently active scalars.
+
+        preference : str, optional
+            When scalars is specified, this is the perfered scalar type to
```

Comment:
```diff
- When scalars is specified, this is the perfered scalar type to
+ When scalars is specified, this is the preferred scalar type to
```

Reply:
That typo was copy-pasted (while it could have very well been mine :) ). I fixed it everywhere in filters.py.